jobserver/migrations/2025_remove_user_github_tokens.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 5.0.8 on 2024-08-19 08:57
+
+from django.db import migrations
+from social_django.models import UserSocialAuth
+
+
+def remove_github_tokens(apps, schema_editor):
+    for user in UserSocialAuth.objects.all():
+        if "access_token" in user.extra_data:
+            del user.extra_data["access_token"]
+        user.save()
```

Comment:
We'd expect `"access_token" in user.extra_data` to be `True` for all users, given `jobserver/auth_pipeline.py`. Nevertheless, we could [lean on Django](https://docs.djangoproject.com/en/5.1/topics/db/queries/#has-key), which I hope leans on the database:

```suggestion
    for user in UserSocialAuth.objects.filter(extra_data__has_key="access_token"):
        del user.extra_data["access_token"]
        user.save()
```